azurepipelines/e2e_test/scenarios/test_runner/scenario_definitions.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------
+#
+# These are the definitions used by the scenario to complete it's operations. It is used by all the TestCases held within
+# the project.
+#
+import os
+import sys
+import uuid
+
+sys.path.append('./')
+from testingtoolkit import UpdateId
+
+#
+# Base Device Definitions
+#
+
+class DuScenarioDefinitionManager:
+    def __init__(self):
+        self.test_device_id = ""
+        self.test_adu_group = ""
```

Comment:
It's better to have these as getter/setter model instead of using direct access in the tests so we can prevent any mistakes when accessing them in the individual tests